### DISCUSSION TITLE
My Application is not getting approved

### DISCUSSION THREAD

**Discussion by ravindu25** [community/community#68027] [GitHub Education]
### Hi everyone,

I used to have access to the Github Student pack for two years now and it expired. However i am still a student (currently in the 3rd year of my 4 year degree program). However, when i try to re-apply, once i upload the Student ID and click submit, i keep on getting this error:
Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.
Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

Even though this says so, I have my name exactly as it appears along with a short bio too. Please help me.

**Comment by Irasil** [comment]
Use the email address of the school where you are studying. This helped me fix the problem.
You can store multiple email addresses on your GitHub profile.

I wish you much success
